### DISCUSSION TITLE
Four line addresses for company and person name

### DISCUSSION THREAD

**Discussion by AiCEG** [manuelbl/SwissQRBill.NET#49] [Ideas]
Hi
In the current version 3.0.3 the address lines are limited to three. Is it intended that this will be extended?
As an example for business addresses with a person name, a four-line address would be necessary. With three lines either the company name or the persons name is lost.
Thank you and best regards.
Aiceg

**Comment by manuelbl** [comment]
The number of address lines is not a limitation of the SwissQRBill.NET library, but rather a limitation of the Swiss QR-bill standard. The data structure simply does not allow more lines (also see https://www.paymentstandards.ch/dam/downloads/ig-qr-bill-en.pdf, ch 4.3.1).

Thus, there are no plans to change it unless the standard changes.

For invoices to businesses, I would omit the person name. While the person name is relevant for sending the invoice and thus is included in the address of the letter and in the invoice header, it's not relevant for the payment part.

**Reply by AiCEG** [reply]
Got it. Thank you for the fast reply.
